### PR TITLE
mozillascience.org redirect rules

### PR DESCRIPTION
**Pull request by cadecairos**
see: https://github.com/MozillaFoundation/mofo-devops/issues/311

I had to implement the blog post redirect as a catch-all, since there wasn't a finite list of blog slugs.

### CODE REVIEW

**Comment by cadecairos** [comment]
@acabunoc can you review this?

File: app/react/main.jsx [hunk 1]
```diff
@@ -58,6 +61,40 @@ render((
       <Route path="resources" component={Resources}/>
       <Route path="style-guide" component={StyleGuide}/>
       <Route path="code-of-conduct" component={CodeOfConduct}/>
+
+      // Redirect old Science site URLs
+      <Redirect from="training" to="programs" />
```
Comment (acabunoc) [inline app/react/main.jsx]: just realized /programs doesn't exist. Can we switch to `resources`?

Comment (cadecairos) [inline app/react/main.jsx]: Isn't it this: https://github.com/mozilla/science.mozilla.org/pull/475/files#diff-e87e7a4271a5c7ae3830035b5e1d1756R60

File: app/react/main.jsx [hunk 2]
```diff
@@ -58,6 +61,40 @@ render((
...
+      <Redirect from="u/:membername" to="/" />
+      <Redirect from="fellows" to="programs/fellowships" />
+      <Redirect from="fellows/2015" to="programs/fellowships" />
+      <Redirect from="training" to="programs" />
+
+      // Event redirects
+      <Redirect from="community-call-jun-09-11-et" to="/programs/events/community-call-jun-09-11-et" />
+      <Redirect from="project-call-may-26" to="/programs/events/project-call-may-26" />
+      <Redirect from="project-call-april-28" to="/programs/events/project-call-april-28" />
+      <Redirect from="space-apps-2016" to="/programs/events/space-apps-2016" />
```
Comment (acabunoc) [inline app/react/main.jsx]: Can we remove this? It's actually a blog post (my bad)

Comment (cadecairos) [inline app/react/main.jsx]: sure can!

File: app/react/main.jsx (hunk 1 above)
Comment (acabunoc) [inline app/react/main.jsx]: But it looks like this:
screen shot 2016-05-31 at 14 38 56

**Comment by acabunoc** [comment]
Just the small change to `resources` and it looks good! 👍

File: app/react/main.jsx (hunk 1 above)
Comment (cadecairos) [inline app/react/main.jsx]: Okay, I'll change the redirect to `resources`. 

Should we file an issue to check what `/progams` and the `Programs` component are doing?

Comment (acabunoc) [inline app/react/main.jsx]: yeah, an issue on that would be a good idea, thanks @cadecairos!

**Comment by cadecairos** [comment]
@acabunoc all fixed up.

Once this is up on prod, I'll flip the switch for mozillascience.org on your signal!

**Comment by acabunoc** [comment]
Ah! getting twitter ready!

**Comment by cadecairos** [comment]
@acabunoc still gotta merge this - also, "the switch" takes 20 minutes to flip lol - no rush required

**Comment by acabunoc** [comment]
Oh, I got super excited...

**Comment by hannahkane** [comment]
🎈 🎈 🎈

**Pull request merged**